cliquet/initialization.py

```diff
@@ -338,11 +338,14 @@ def initialize(config, version=None, project_name=None, default_settings=None):
         in application settings.
     :param dict default_settings: Override cliquet default settings values.
     """
-    if default_settings:
-        config.add_settings(default_settings)
-
     settings = config.get_settings()
 
+    if default_settings:
+        for key, value in default_settings.items():
+            if not settings.get(key):
```

Comment:
Back to the original spec of this ``default_settings``: «*Override cliquet default settings values*»

It could become something like this:
```python
for key, value in default_settings.items():
    if settings[key] == cliquet.DEFAULT_SETTINGS[key]:
        # Override only if not set by user.
        settings[key]  = value
```